run_docker.sh

```diff
@@ -1 +0,0 @@
-docker run -ti --rm -v $(pwd):/Qt.py --entrypoint bash mottosso/qt.py27
```

Comment:
Sure. But I don't quite understand what the container is executing with that entrypoint.

Could you give me the exact command(s) you'd like to see in `run_docker.sh`?